source/declarations.tex

```diff
@@ -4328,10 +4328,11 @@
 ambiguous, the initialization is ill-formed.
 \item
 Otherwise (i.e., for the remaining copy-initialization cases),
-user-defined conversion sequences that can convert from the
+user-defined conversion functions and converting constructors that can convert from the
```

Comment:
The usual phrase to use here is simply "user-defined conversions", defined in [class.conv]. That's also what [over.match.copy] talks about.